trieste/__init__.py

```diff
@@ -16,6 +16,11 @@
 models (see :mod:`~trieste.models`), and data sets (see :mod:`~trieste.data`). The
 :mod:`~trieste.acquisition` package provides a selection of acquisition algorithms and the
 functionality to define your own.
+
+Bibliography
+============
+
+.. bibliography::
```

Comment:
i'm not sure where is best to put the bibliography
- relevant references in each module
- in the library root (as I've done here)
- on the home page
- in a separate "bibliography" page (I couldn't get this to work)

Reply:
i think in the library root is OK for now. We can move it later if and when we think of somewhere